20221111_homework/model_append.py

```diff
@@ -0,0 +1,16 @@
+def finding_previvios_id():
+    with open("phone_book.csv", "r", encoding="utf-8") as data:
+        for line in data:
+            pass
```

Comment:
Зачем же гонять для этого цикл?
```suggestion
        line = data.readlines()[-1]
```